slave/executors.py

```diff
@@ -132,11 +134,11 @@ def execute(self, benchmark, env, args, profile):
         # delete profile
         runner.rm("profile/")
 
-        # create new profile
-        runner.mkdir("profile/")
+        # Force stylo off.
+        prefs["layout.css.servo.enabled"] = False
```

Comment:
one thing i'd maybe suggest is if we can pull this out into a separate file so it's just a case of listing out prefs and desired settings (presuming we have many more experiments in the future that we might want some control over)... there's also maybe further discussion about using a service to set our automation to the correct state (maybe even allowing remote flipping so we can effectively a/b our results)